### ISSUE TITLE
Image upload / max_size only 1MB

### ISSUE THREAD

**Issue by Paddy0174** [hassio-addons/addon-bookstack#11]
# Problem/Motivation
I'm trying to upload a file with a size more than 1MB. The upload seems to be canceled by, my guess, NGINX because of the `max_upload_size`.

Unfortunately even with `log_level: debug` I don't get any errors in the log. This is what the log shows (seems to be ok, with a response code 200)
```
[15/Mar/2020:15:04:00 +0100] 200 -(192.168.178.23) GET /uploads/images/user/2020-03/thumbs-40-40/Steffi-avatar.png HTTP/1.1 (Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:74.0) Gecko/20100101 Firefox/74.0)
```

## Expected behavior
Upload doesn't fail. :)

## Actual behavior
Upload fails

## Steps to reproduce
Edit (or setup new) page in a book and try to put in an image with a size bigger than 1MB. It doesn't matter which way you prefer, it fails either way.

## Proposed changes
My guess is the "culprit" is the NGINX max_upload_size config value. Maybe we could get this into the configuration, so it would be changeable for the user of the addOn.

**Comment by sinclairpaul** [comment]
It's the php config I believe, I will get it fixed up in a bit.

**Comment by sinclairpaul** [comment]
Fixed in v0.2.1, if you refresh the addon store you should see the update.

**Comment by Paddy0174** [comment]
Lightning fast response, thanks a lot! Oh, and yes, it's working! 🥇
